java2typescript-jackson/README.md

```diff
@@ -68,7 +68,14 @@ export type MyEnum = "VAL1" | "VAl2" | "VAL3";
 ```
 
 This will keep strong-typing for the values, while allowing for string literal values of enum properties. Take a look at
-the [test that turns on this preference](src/test/java/java2typescript/jackson/module/WriterPreferencesTest.java#L67)
+the [test that turns on this preference](src/test/java/java2typescript/jackson/module/WriterPreferencesTest.java#L67).
+
+There is an additional option you can set on the writer which, if enabled, generates typescript constants which mirror the enum values. To enable this, do the following:
+
+```Java
+mWriter.preferences.useEnumPattern();
+mWriter.preferences.createConstantsForStringLiteralTypeEnums();
```

Comment:
i'd avoid introducing new method to the preferences, that can only be used in conjunction with useStringLiteralTypeForEnums(). I'd either overload `useStringLiteralTypeForEnums()` or rather change it: `useStringLiteralTypeForEnums(boolean withConstants)`. 

I think in this case it makes more sense to change it by adding the boolean (in this case when updating to the latest, there would also be a compile-time error with java2typescript-jacson usage, not unexpected change in its output).